src/Templating/src/Microsoft.DotNet.Web.ProjectTemplates/content/EmptyWeb-CSharp/Program.cs

```diff
@@ -14,11 +15,14 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            CreateWebHostBuilder(args).Build().Run();
+            CreateHostBuilder(args).Build().Run();
         }
 
-        public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseStartup<Startup>();
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
```

Comment:
Still not 100% sold on `Host`. Perhaps `AppHost`?

Reply:
Yea Host is hella generic but this is the generic host 😬. We should do an experience review.

Reply:
AppHost I don’t like but that isn’t a hill I would die on.

- Host
- AppHost
- ApplicationHost
- GenericHost